.env.prod

```diff
@@ -2,22 +2,22 @@ TESTFAIRY_ENABLED=0
 STUB_STATUS_GO=0
 ETHEREUM_DEV_CLUSTER=0
 MAINNET_WARNING_ENABLED=1
+OFFLINE_INBOX_ENABLED=1
 RPC_NETWORKS_ONLY=1
-OFFLINE_INBOX_ENABLED=0
 LOG_LEVEL=info
 LOG_LEVEL_STATUS_GO=info
-JSC_ENABLED=1
 QUEUE_MESSAGE_ENABLED=0
 RN_BRIDGE_THRESHOLD_WARNINGS=0
-MIXPANEL_TOKEN=2584e00100d319d12e538cc4d0fa9fc1
-TESTFAIRY_TOKEN=969f6c921cb435cea1d41d1ea3f5b247d6026d55
 POW_TARGET=0.002
 POW_TIME=1
+MIXPANEL_TOKEN=2584e00100d319d12e538cc4d0fa9fc1
 DEFAULT_NETWORK=testnet_rpc
+TESTFAIRY_TOKEN=969f6c921cb435cea1d41d1ea3f5b247d6026d55
 INSTABUG_TOKEN=758630ed52864cbad9c5eeeac596c60c
 DEBUG_WEBVIEW=0
 INSTABUG_SURVEYS=1
 GROUP_CHATS_ENABLED=0
 FORCE_SENT_RECEIVED_TRACKING=0
 USE_SYM_KEY=0
+JSC_ENABLED=1
```

Comment:
I thought we removed JSC_ENABLED flag?

Reply:
no idea, I just moved it down :)

Reply:
(so when you look at `.env` and `.env.prod` side-by-side, it is useful :)

Reply:
https://github.com/status-im/status-react/pull/4051/files

Reply:
for some reason it was removed only in .env

Reply:
cool, I'll remove it then